diff --git a/pkgs/games/path-of-building/default.nix b/pkgs/games/path-of-building/default.nix
index 753001d192b9..9d7cf5088be3 100644
--- a/pkgs/games/path-of-building/default.nix
+++ b/pkgs/games/path-of-building/default.nix
@@ -17,13 +17,13 @@
 let
   data = stdenv.mkDerivation (finalAttrs: {
     pname = "path-of-building-data";
-    version = "2.52.3";
+    version = "beta";
 
     src = fetchFromGitHub {
       owner = "PathOfBuildingCommunity";
       repo = "PathOfBuilding";
-      rev = "v${finalAttrs.version}";
-      hash = "sha256-hwluSZGYRMN32dSLIe8Cs6l6R84V5fNCuHYtoc7+b00=";
+      rev = "beta";
+      hash = "sha256-vVb8Hs6R+WfczMyJeL6cDgIfjVn23hZWPVXC8g+Z+oQ=";
     };
 
     nativeBuildInputs = [ unzip ];
